pages/measure-conformance.md

```diff
@@ -271,9 +275,9 @@ Snippet 3-7: Example of [effectivePeriodAnchor extension](StructureDefinition-cq
 
 **Conformance Requirement 3 (Measurement Period):** [<img src="assets/images/conformance.png" width="20" class="self-link" height="20"/>](#conformance-requirement-3)
 {: #conformance-requirement-3}
-1. The value of the "Measurement Period" control variable SHALL be made available to CQL libraries as the value of the effectivePeriod element.
-2. CQL libraries that require access to the "Measurement Period" control variable SHALL either declare the type of the "Measurement Period" parameter as an interval of DateTime or provide a default value as an interval of DateTime.
-3. The value of the "Measurement Period" control variable MAY be made available using the `cqfm-effectivePeriodAnchor` and `cqfm-effectivePeriodDuration` extensions to specify the length of the reporting period, anchored to a first effective start date.
+1. FHIR-based eCQMs SHALL provide either an `effectivePeriod` element, or an `cqfm-effectivePeriodAnchor` and `cqfm-effectivePeriodDuration` extension
+2. Measurement Period SHALL be either the `effectivePeriod` as specified, or an appropriate interval of length duration, starting at the specified anchor
```

Comment:
> Measurement Period SHALL be ... an appropriate interval of length duration, starting at the specified anchor

hey @ar-esacinc, I found this commit from your link on FHIR-27751. If I understand correctly, the start of the Measurement Period can be _any_ date, as long is it's **greater than or equal to** the `cqfm-effectivePeriodAnchor`. I'm basing this off Bryn's comments in zulip and on the ticket:

> For example, a measure anchored to January 1, 2019 with a calendar duration of 1 year, would have valid Measurement Periods of 1/1/2019-12/31/2019, 1/1/2020-12/31/2020, etc.

It might be worth including this example to clarify. thanks!

Reply:
hi @mashton, the clarification has been added to line 259.